docs/root/configuration/http/http_conn_man/response_code_details.rst

```diff
@@ -0,0 +1,93 @@
+.. _config_http_conn_man_details:
+
+Response Code Details
+=====================
+
+If _%RESPONSE_CODE_DETAILS%_ is configured on in access logging, every log entry should include the detailed reason that stream ended.
```

Comment:
Ref link to the access logging formatter docs?

Also, should we link to the local reply mapper where I think this can also be used and maybe link to here from there?